app/views/pages/_blog_search_form.html.haml

```diff
@@ -0,0 +1,3 @@
+= simple_form_for blog_search, url: blog_search_path, method: :post, remote: true do |f|
```

Comment:
Good to see someone using `remote: :true` for stuff! (Here, and with comment creation.)